SystemInformer/prpgtok.c

```diff
@@ -22,15 +23,21 @@ NTSTATUS NTAPI PhpOpenProcessTokenForPage(
     NTSTATUS status;
     HANDLE processHandle;
 
-    if (!Context)
+    if (!Context || !PH_IS_REAL_PROCESS_ID(Context))
```

Comment:
Is this and the next lines ok? i thought, we should make this sync how in procprv.c ->` PhpFillProcessItem()`

Reply:
I don't see anything wrong with this, it's a private API.